src/primitives/_button.scss

```diff
@@ -15,9 +15,7 @@
   text-align: center;
   text-decoration: none;
 
-  @include device.min-width(550px) {
-    @include padding.inline('dog');
-  }
+  @include padding.inline('dog');
```

Comment:
I'm being picky, I know, but can you put this in relative alphabetical order with the other rules? I went to the effort of enabling us to do that in the stylelint config, so we should do it if we can ;)